docs/pages/admin-guides/access-controls/access-lists/nested-access-lists.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Nested Access Lists
+description: Learn how to use nested Access Lists to manage complex permissions and grant inheritance in Teleport.
+---
+
+This guide will help you:
+
+- Understand how nesting and inheritance work in Access Lists
+- Create a nested Access List
+- Verify inherited permissions granted through the nested Access List
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+- A user with the `editor` role or equivalent permissions to create and manage Access Lists.
+- Familiarity with basic Access List concepts (see the [Getting Started with Access Lists guide](./guide.mdx)).
+- At least one user with only the `requester` to add to the Access List.
+- At least one application or resource to grant access to.
+
+## What Are Nested Access Lists?
+
+Nested Access Lists allow inclusion of an Access List as a member or owner of another Access List.
+This enables hierarchical permission structures where permissions can be inherited from multiple levels of
+parent Access Lists.
+
+### How Inheritance Works
+
+- **Membership Inheritance**: If Access List B is added as a member of Access List A,
+all users who are members of Access List B inherit Access List A's member grants (roles and traits).
+- **Ownership Inheritance**: If Access List B is added as an owner of Access List A,
+all users who are members of Access List B inherit Access List A's owner grants, and
+can perform owner actions on Access List A, such as modifying it or managing its members.
+
+Inheritance is recursive – members of a child Access List can themselves be Access Lists
+with their own members, and so on. However, circular nesting is not allowed, and nesting is limited
+to a maximum depth of 10 levels.
+
+For more information, see the [Access Lists reference](../../reference/access-controls/access-lists.mdx).
+
+## Creating a Nested Access List
+
+Let's walk through an example of creating a nested Access List and establishing inheritance.
+
+### Step 1/4. Create Parent Access List
+
+In the Teleport Web UI, go to the "Identity" tab and select "Access Lists" from the sidebar.
+Click on "Create New Access List", and fill in the details for the parent Access List. For example:
+
+- **Title**: Parent Access List
+- **Deadline for First Review**: Select a future date.
+- **Member Grants**: Add roles or traits that you want members to inherit, such as the `access` role.
+- **Owners**: Add yourself or any appropriate users as owners.
+- **Members**: Leave this empty for now, as we'll add a child Access List as a member later.
+
+Click "Create Access List" to save the new Access List.
+
+### Step 2/4. Create Child Access List
```

Comment:
Why don't we reorder these steps and create the child AL first, and then add it as a member right when creating the parent list? This way the guide would have 1 less step.